lib/release_notes_bot/projects.ex

```diff
@@ -100,19 +100,6 @@ defmodule ReleaseNotesBot.Projects do
     params
   end
 
-  # parse_action can reduced to regular pattern matching in calling function
```

Comment:
@zach-bob I see what's going on here, but perhaps it could be better explained in the commit message.  When I see "[feat] create slack modal interaction controller", I don't expect to see so many deleted lines from other files.  Mentioning that you're extracting some behavior from a previous controller would help convey intent.

Reply:
Right, good catch. I have renamed to: `[feat] extract from captains controller to new modal interaction controller`.